client/src/components/HistoryViewer/HistoryViewerVersionList.js

```diff
@@ -82,6 +94,8 @@ HistoryViewerVersionList.propTypes = {
   messages: PropTypes.arrayOf(messageType),
   VersionComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
   versions: PropTypes.arrayOf(versionType),
+  compareFrom: versionType,
+  compareTo: versionType,
```

Comment:
You mentioned in Slack you were going to update this to an object, would be cool to do now before we merge more individual props

Reply:
I was hoping we'd get this in and then I'll touch it in the PR to fix up the object. Then I can add a prop type similar to how version currently is... It's a bit awkward trying to implement it in multiple PRs.